scripts/setup.sh

```diff
@@ -200,9 +200,10 @@ elif [ "$VAGRANT" = true ]; then
   export XVFB_PREFIX="/usr/bin/xvfb-run"
   # Enforce proper ownership on oppia, oppia_tools, and node_modules or else NPM installs will fail.
   sudo chown -R vagrant.vagrant /home/vagrant/oppia /home/vagrant/oppia_tools /home/vagrant/node_modules
-elif [ -f "/usr/bin/google-chrome" ]; then
+
+elif [ -f "/usr/bin/chromium-browser" ]; then
   # Unix.
-  export CHROME_BIN="/usr/bin/google-chrome"
+  export CHROME_BIN="/usr/bin/chromium-browser"
```

Comment:
The point still stands -- you can't do something that breaks things for other developers. I'm fine with putting the chromium check before chrome, but not removing chrome altogether.

Also, with regards to the Chrome issue you encountered, could you please file an issue on our issue tracker and attach logs/screenshots? We might need to investigate it. Thanks!